### DISCUSSION TITLE
Cancelling a `PerformFromMenuRunner` action should not trigger a notification in production releases

### DISCUSSION THREAD

**Discussion by frenzibyte** [ppy/osu#26305] [Ideas]
### Type

Game behaviour

### Bug description

Such notifications should not appear to the end user, they're completely technical and do not affect the user in any way.

### Screenshots or videos

The notification in question:

https://github.com/ppy/osu/assets/22781491/dd8cdaa4-d45d-4d65-a284-d69f53edc1ce

### Version

-

### Logs

-

**Comment by peppy** [comment]
Disagree
